pkg/dbop/status_test.go

```diff
@@ -129,7 +129,9 @@ var _ = Describe("status", func() {
 		st1, err = ops[1].GetStatus(ctx)
 		Expect(err).NotTo(HaveOccurred())
 		Expect(st0.GlobalVariables.ExecutedGTID).To(Equal(st1.GlobalVariables.ExecutedGTID))
-		Expect(st0.GlobalStatus.SemiSyncMasterWaitSessions).NotTo(Equal(0))
+		Eventually(func() {
```

Comment:
Isn't it necessary to update ops[0].GetStatus(ctx)?
And it seems better to use Eventually together with Should or ShouldNot.

```suggestion
        Eventually(func() int {
            st0, _ := ops[0].GetStatus(ctx)
            return st0.GlobalStatus.SemiSyncMasterWaitSessions
        }).ShouldNot(Equal(0))
```